lua/indent-tools/health.lua

```diff
@@ -1,5 +1,5 @@
 local M = {}
-local health = vim.health
+local health = vim.health or require "health"
```

Comment:
In order to make it consistent with the rest of the code can you change this to this line please?
```lua
local health = vim.health or require("health")
```

Reply:
Okay, all done!